Many enhancements + support chrome 99

Blondy314 opened the pull request.

yairp03 commented on src/DownloadForm.Designer.cs:
```diff
@@ -43,6 +43,7 @@ private void InitializeComponent()
             this.OverallProgress_Label = new Guna.UI2.WinForms.Guna2HtmlLabel();
             this.EpisodeLoad_Label = new Guna.UI2.WinForms.Guna2HtmlLabel();
             this.EpisodeDonwload_Label = new Guna.UI2.WinForms.Guna2HtmlLabel();
+            this.txtUrl = new System.Windows.Forms.TextBox();
```
The URL alone without the cookie is not worth much, I don't know if there's a point in adding it.

Blondy314 replied:
Removed

yairp03 reviewed (changes requested):
The URL on the download form is a bit useless on its own, I think it should be removed or you can find a way to make it useful.
Besides that, looks great! Can't wait to merge this PR.
Glad you helping me revive this old project ;)

Blondy314 commented:
Hi,

Removed Download URL and added a few more improvements, hope u like it and
merge :)

Inbar.

On Mon, Mar 28, 2022 at 1:55 AM Yair Peretz ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> The URL on the download form is a bit useless on its own, I think it
> should be removed or you can find a way to make it useful.
> Besides that, looks great! Can't wait to merge this PR.
> Glad you helping me revive this old project ;)
> ------------------------------
>
> In src/DownloadForm.Designer.cs
> <https://github.com/yairp03/SdarotTV-Downloader/pull/3#discussion_r835974132>
> :
>
> > @@ -43,6 +43,7 @@ private void InitializeComponent()
>              this.OverallProgress_Label = new Guna.UI2.WinForms.Guna2HtmlLabel();
>              this.EpisodeLoad_Label = new Guna.UI2.WinForms.Guna2HtmlLabel();
>              this.EpisodeDonwload_Label = new Guna.UI2.WinForms.Guna2HtmlLabel();
> +            this.txtUrl = new System.Windows.Forms.TextBox();
>
> The URL alone without the cookie is not worth much, I don't know if
> there's a point in adding it.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/yairp03/SdarotTV-Downloader/pull/3#pullrequestreview-922507564>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFHD6FX6Z4DZMVXIFUD7WDDVCDRPTANCNFSM5RXOTYMQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

Blondy314 commented:
Any update on merging this?

Blondy314 commented:
Hi, pinging on this :)

yairp03 commented:
Hi, sorry for the late reply.
I've been busy lately, so I haven't had the chance to review the changes,
hopefully I can do so this week and approve the PR.

On Mon, Apr 11, 2022, 14:34 Blondy314 ***@***.***> wrote:

> Hi, pinging on this :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/yairp03/SdarotTV-Downloader/pull/3#issuecomment-1094941653>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AK3V2MY6NIDCEOX5Q6RGICLVEQE4TANCNFSM5RXOTYMQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

Blondy314 commented:
Pinging again, before they close the site ;)
